### ISSUE TITLE
How can the segmentation network discriminate different lanes?

### ISSUE THREAD

**Issue by swpsgithub** [fabvio/Cascade-LD#9]
Hi, @fabvio 
Thanks for your great work. I've met some problems about training my own network. The main problem is how can the segmentation network discriminate different lanes. I'm trying the position of the lane to tell the difference which only leads the segmentation network learn other things. Did you use some post-processing methods? 
Looking forward to your reply.

**Comment by fabvio** [comment]
Hi, thanks for your interest. To answer your question, we do not do any post-processing. In the paper it is described a training procedure that I will briefly summarize here. First, you train a segmentation network using binary labels (lane/background) with a standard cross-entropy loss. Then, you finetune the obtained network with the [provided clustering loss function](https://github.com/fabvio/Cascade-LD/blob/master/loss.py), where you use as labels for each lane a different ID.

**Comment by swpsgithub** [comment]
My bad! Reading the paper not carefully. Thanks for your kind reply.
